collection-scripts/gather_kas_startup_termination_logs

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Downloads the startup.log and termination.log (and its rotated copies) from /var/logs/kube-apiserver on each master node
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+BASE_COLLECTION_PATH="${BASE_COLLECTION_PATH:-/must-gather}"
+
+echo "WARNING: Collecting one or more kube-apiserver related logs on ALL masters in your cluster. This could take a large amount of time." >&2
+
+# the command executed by xargs below expects four parameters:
+# $1 - node path under /var/logs to download
+# $2 - local output path
+# $3 - node name
+# $4 - log file name
+paths=(kube-apiserver)
+for path in "${paths[@]}" ; do
+  output_dir="${BASE_COLLECTION_PATH}/static-pods/$path"
+  mkdir -p "$output_dir"
+  oc adm node-logs --role=master --path="$path" | \
```

Comment:
@p0lyn0mial please add a note pointing at https://github.com/openshift/library-go/pull/1196/files#diff-387eea59f06b61f1bf3cce6c2fd89f63150b60bfee719c53efa885a87bbc5be9R180 that explains how much data this should be collecting. 

@sttts and I discussed this on slack; thus it should be ~3 pod logs * ~3 (MaxFiles) * ~100 KB +/- (a bit depending on the math). 

I simply want to capture how much we expected this to retain.